src/test/java/page/AccountListPage.java

```diff
@@ -0,0 +1,45 @@
+package page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+import static tests_data.TestData.account;
+
+public class AccountListPage extends BasePage {
+    public AccountListPage(WebDriver driver) {
+        super(driver);
+    }
+
+    By NEW_BUTTON = By.cssSelector("[title=New]");
+    String NAME_ACCOUNT = account.getAccountName().split(" ")[0];
+    By TITLE_NAME = By.cssSelector("[title~=" + NAME_ACCOUNT + "]");
+
+    @Override
+    public AccountListPage waitForPageOpened() {
```

Comment:
if you're not using 'chain of invocations', make this method return void (and other waiters as well)

Reply:
fix